app/views/teacher_mailer/initial_checks_passed.text.erb

```diff
@@ -0,0 +1,25 @@
+Dear <%= application_form_full_name(@application_form) %>
+
+We’ve performed our initial checks on your application.
+
+<%= render "shared/teacher_mailer/reference_number" %>
+
+You should now request your <%= @teaching_authority_certificate %> from <%= @teaching_authority_name %>. Contact them directly and instruct them to send the document to ApplyQTS.Verification@education.gov.uk.
```

Comment:
The verification email is defined in the locale file:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/2938ba3a3916335f944b6ce288a4bceba5bacafb/config/locales/en.yml#L16

Reply:
Easy to forget! :D